packages/@aws-cdk/aws-certificatemanager/README.md

```diff
@@ -100,15 +100,40 @@ new acm.Certificate(this, 'Certificate', {
 ## Cross-region Certificates
 
 ACM certificates that are used with CloudFront -- or higher-level constructs which rely on CloudFront -- must be in the `us-east-1` region.
-The `DnsValidatedCertificate` construct exists to facilitate creating these certificates cross-region. This resource can only be used with
-Route53-based DNS validation.
+CloudFormation allows you to create a Stack with a CloudFront distribution in any region. In order
+to create an ACM certificate in us-east-1 and reference it in a CloudFront distribution is a
+different region, it is recommended to perform a multi stack deployment.
+
+Enable the Stack property `crossRegionReferences`
+in order to access the cross stack/region certificate.
+
+> **This feature is currently experimental**
 
 ```ts
-declare const myHostedZone: route53.HostedZone;
-new acm.DnsValidatedCertificate(this, 'CrossRegionCertificate', {
-  domainName: 'hello.example.com',
-  hostedZone: myHostedZone,
-  region: 'us-east-1',
+const stack1 = new Stack(app, 'Stack1', {
+  env: {
+    region: 'us-east-1',
+  },
+  crossRegionReferences: true,
+});
+const cert = new acm.Certificate(east1Stack, 'Cert', {
```

Comment:
Is this supposed to be scoped to `stack1` or am I missing where `east1Stack` is defined?

Reply:
Ahh good catch, I copied this example from somewhere else and missed that update.